xicam/core/data/__init__.py

```diff
@@ -26,15 +23,13 @@ def detect_mimetypes(filename: str) -> List[str]:
         # future if we discover reason to. Therefore, sniffers should not
         # assume that they will receive this exact number of bytes.
         first_bytes = file.read(64)
-    for ep in entrypoints.get_group_all("databroker.sniffers"):
-        try:
-            content_sniffer = ep.load()  # TODO: only load sniffers once
-        except Exception as ex:
-            msg.logError(ex)
-        else:
-            matched_mimetype = content_sniffer(filename, first_bytes)
-            if matched_mimetype:
-                matched_mimetypes.append(matched_mimetype)
+
+    from xicam.plugins import manager as plugin_manager
```

Comment:
i like that you are deferring to the plugin manager here instead of directly using the entrypoint API